custom_components/spotcast/__init__.py

```diff
@@ -10,10 +10,12 @@
 from homeassistant.core import callback
 from homeassistant.exceptions import HomeAssistantError
 import homeassistant.helpers.config_validation as cv
-from homeassistant.components.cast.media_player import KNOWN_CHROMECAST_INFO_KEY
+from homeassistant.components.cast.media_player import CastDevice
 from homeassistant.components.cast.helpers import ChromeCastZeroconf
+from homeassistant.components.spotify.media_player import SpotifyMediaPlayer
+from homeassistant.helpers import entity_platform
 
-__VERSION__ = "3.4.7"
+__VERSION__ = "3.5.2"
```

Comment:
It's now inconsistent with `manifest.json`. I think it's better to drop it and keep version only in `manifest.json`.